js/apps/admin-ui/src/identity-providers/add/AdvancedSettings.tsx

```diff
@@ -125,6 +136,90 @@ export const AdvancedSettings = ({ isOIDC, isSAML }: AdvancedSettingsProps) => {
       />
       <SwitchField field="config.hideOnLoginPage" label="hideOnLoginPage" />
 
+      <FormGroupField label="filteredByClaim">
```

Comment:
The feature is implemented just for OIDC Identity providers. So are we showing the relevant options just for OIDC Identity providers? As I think it should be hidden for SAML IDP and also for Social Identity providers, which are not OIDC Identity provider.

Reply:
Good catch, fixing and revalidating it